sigopt/cli/commands/training_run/base.py

```diff
@@ -0,0 +1,6 @@
+from ..base import sigopt_cli
+
+
+@sigopt_cli.group("training_run")
+def training_run_command():
+  '''Commands for managing SigOpt TrainingRuns.'''
```

Comment:
prefer the name "run" instead of "training run". however, "sigopt run" is already the name of a command

Reply:
Renamed to runs.